sparkle/gui/main_control.py

```diff
@@ -988,9 +990,12 @@ def tabChanged(self, tabIndex):
             self.ui.aifsSpnbx.setValue(self.acqmodel.calibration_genrate())
             self.ui.aifsSpnbx.setEnabled(False)
             self.setCalibrationDuration()
+            self.ui.mphoneCalBtn.setText('Calibrate Mic')
```

Comment:
Because the `mphoneCalBtn` text will never need to change you can change the text in the .ui file (main_control.ui) using QtDesigner. This means that the auto generated `main_control_form.py` file will also need to be updated (using the pyuic4 utility).